docs/tutorial-solver.md

```diff
@@ -162,3 +151,62 @@ To silence these warnings and for an overall cleaner logging experience, you may
+
+_Note that_ the default subsidy factor is 1.
+
+2. **Static:**
+   Uses `static_min_avg_fee_per_order` and `static_max_gas_price` (in base units - wei).
+   Using this strategy with certain configuration values can be dangerous.
+   For example, if there are many overlapping open orders for low valued trades and the `static_min_avg_fee_per_order` is set to zero.
+   This would mean that running a solver with this EV-strategy would blindly match orders for substantially low reward.
+   Similarily, if the `static_max_gas_price` is set too high and you have a solver running during times with high network demand/congestion, gas prices could increase to the point that your solver service submits very expensive transactions.
+
+3. **Combined:**
+   This approach uses a hybrid of Dynamic bounded by Static.
+   That is,it evaluates and Dynamic with priority.
+   If the dynamic evaluation fails or the result is worse (i.e. larger `min-avg-fee` or lower `max-gas-price`) then falls back to the static evalaution instead.
+   For somewhat more clarity, the EV-computer deems lower minimum average fee and higher max gas price as "better" so to say that the `min-avg-fee` is bounded _above_ by static evaluation with a preference for the lower value and the `max-gas-price` is bounded _below_ by the static with a preference for the higher.
+
+#### Example EV Configurations
+
+1. Dynamic - default: Without setting any economic viability arguments, a solver will default to Dynamic with a subsidy factor of 1. This means that solutions will only be submittted if the estimated transaction cost is at least equal to the fee-reward.
+
+2. Static with no min average fee and 100 GWei max gas price: This strategy would only prevent a solver from submitting solutions if the current fast-gas price is greater than 100 GWei.
+   This could be considered dangerous for reasons describbed above, but may also be beneficial if the solver themselves has an overlapping open order that would yeild a valuable trade. Such a configuration would require the following env vars be set.
+   Note that; if the `ECONOMIC_VIABILITY_STRATEGY` is set to static, then both of the following two _must_ be specified.
+
+```sh
+ECONOMIC_VIABILITY_STRATEGY=static
+STATIC_MIN_AVG_FEE_PER_ORDER=0
+STATIC_MAX_GAS_PRICE=100000000000
+```
+
+3. Combined with min fee of 10 USD, max gas of 40 GWei and subsidy factor of 1/2: Since this is a hybrid of both Dynamic and Static, one must specify the static environment variables, but is left to choose the if subsidy factor differs from the default of 1.
+
+```sh
+ECONOMIC_VIABILITY_STRATEGY=combined
+STATIC_MIN_AVG_FEE_PER_ORDER=10
+STATIC_MAX_GAS_PRICE=40000000000
+ECONOMIC_VIABILITY_SUBSIDY_FACTOR=0.5
+```
+
+#### Examples of Costly Solutions
```

Comment:
I wouldn't include this section. What is the value of showing these "stupid looking transactions" to the customers of GP? 
While those specific examples seem to be mistakes on our side, there could be an expensive solution with little fee that is worthwhile because of some good arbitrage opportunity for the person submitting the tx. 
I think this query could be interesting for us internally to see how much we subsidize solution submission.

Reply:
All fixed up!